dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -59,9 +56,7 @@ def create
     render head :bad_request unless section
 
     # TODO: Move to an after_create step on Section model when old API is fully deprecated
-    if script_to_assign
-      current_user.assign_script script_to_assign
-    end
+    current_user.assign_script @unit if @unit
```

Comment:
is the comment still valid? which old API are we talking about?

Reply:
We have two sections_controllers. One in /api/vi and one that is not. I did not pull on the threads of why we have two but it was not necessary to update the other controller to make things work here.

Reply:
I asked about it in this slack thread: https://codedotorg.slack.com/archives/C0T10HG6N/p1646679987426589